packages/flutter/lib/src/widgets/pages.dart

```diff
@@ -2,6 +2,10 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+// TODO(chunhtai): remove ignoring deprecated member use analysis
+// when PageRouteBuilder.pageBuilder parameter is removed. See
+// ignore_for_file: deprecated_member_use_from_same_package
```

Comment:
please use explicit //ignores everywhere so we don't accidentally ignore unrelated things